developer-experience/readme.md

```diff
@@ -0,0 +1,203 @@
+
+## Goals
+
+- Maximize the amount of time engineers spend on writing code that fulfills story acceptance and done-done criteria.
+- Minimize the amount of time spent manual setup and configuration of tooling
+- Minimize regressions and new defects by making end-to-end testing easy
+
+## Impact
+
+Developer experience can have a significant impact on the efficiency of the day to day execution of the team. A positive experience can pay dividends throughout the lifetime of the project; especially as new developers join the team.
+
+- Increased Velocity - Team spends less time on non-value-add activities such as dev/local environment setup, waiting on remote environments to test, and rework (fixing defects).
+- Improved Quality - When its easy to debug and test, developers will do more of it. This will translate to fewer defects being introduced.
+- Easier Onboarding & Adoption - When dev essential tasks are automated, there is less documentation to write and, subsequently, less to read to get started!
+
+**Most importantly, the customer will continue to accrue these benefits long after the code-with engagement.**
+
+## Measures
+
+### Time to First E2E Result (aka F5 Contract)
+
+Assuming a laptop/pc that has never run the solution, how long does it take to setup and run the whole system end-to-end and see a result.
+
+### Time To First Commit
+
+How long does it take to make a change that can be verified/tested locally. A locally verified/tested change is one that passes test cases without introducing regression or breaking changes.
+
+## Participation
+
+Providing a positive developer experience is a team effort. However, certain members can take ownership of different areas to help hold the entire team accountable.
+
+### Tech Lead - Set the bar
+
+The following are examples of how the Tech Lead might set the bar for dev experience
+
+- Determines development environment (IDE, hosting, etc)
+- Determines source control environment and number of repo's required
+- Given development environment and repo structure sets expectations for team to meet in terms of steps to perform the essential dev tasks
+- Nominates the DevEx Champion
+
+### DevEx Champion - Identify Iterative Improvements
+
+The devex champion takes ownership in holding the team accountable for providing a positive developer experience. The following outline responsibilities for the DevEx champion.
+
+- Actively seek opportunities for improving the solution developer experience
+- Work with the tech lead to iteratively improve team expectations for developer experience
+- Curate a backlog actionable stories that identify areas for improvement.
+- Serve as subject matter expert for the rest of the team. Help the team determine how to implement devex expectations and identify deviations.
+
+### Senior Members - Assert Expectations

```

Comment:
While I imagine that somebody with a "senior" title may do this more proactively, I think that non "senior" members hold a very valuable perspective here as well.

Reply:
Agree, i think we got the ownership aspect @jplane was after with the first two headings (tech lead and devex champion)

Reply:
changing to team members rather than senior...